### PR TITLE
10 yvers v2

### PR DESCRIPTION
**Pull request by depeele**
Changed yvers parsing to allow top-level formatting blocks that contain additional sub-block and character formatting elements.

This allows multiple verses to exist within a single top-level formatting block enabling block-level formatting (e.g. indents, text alignment).

Created a new ChapterYvers component to handle rendering this new format.

### CODE REVIEW

File: web-admin/etc/yvers-parse-version-2 [hunk 1]
```diff
@@ -215,7 +216,7 @@ function _parseBook( abbrev, chapters, ref_filter = null ) {
     }
   });
   
-  console.log('%s', Inspect( book_json, 8, !'colors' ));
+  console.log('%s', Inspect( book_json, 10, !'colors' ));
```
Comment (oparkins) [inline web-admin/etc/yvers-parse-version-2:219]: Magic number alert

File: web-ui/src/lib/VerseInterlinear.svelte [hunk 2]
```diff
@@ -21,11 +22,12 @@
 {/if}
 
 <div class='inline-flex flex-wrap'>
-  <p class="pt-[1em]">{verse_num}</p>
+  <p class='pt-[1em]'>{verse_num}</p>
   {#each verse.markup as markup, m_dex}
     {#if markup.text !== undefined}
-      <WordInterlinear word={markup} word_ref={verse_ref.replaceAll('.','-') + m_dex} />
+      {@const word_ref = verse_ref.replaceAll('.','-') +'-'+ m_dex }
```
Comment (oparkins) [inline web-ui/src/lib/VerseInterlinear.svelte:28]: Oh that is a nice feature of Svelte!

File: web-ui/etc/config.yaml [hunk 3]
```diff
@@ -4,10 +4,8 @@ verbosity:  1
...
   port: 5173
-# } web_ui
 
-# Access to web-services (REST and JsonRPC)
-web_services: # {
-  host: api.ayia.nibious.com
-  port: 443
-#} web_services
+  api_map: #{
+    versions: http://localhost:4000/api/v2/versions
```
Comment (oparkins) [inline web-ui/etc/config.yaml:9]: Should this be put back to `api.ayia.nibious.com`?

File: web-admin/yvers-parsing.md [hunk 4]
```diff
@@ -1,109 +1,194 @@
...
-     block (e.g. s1, s2, p, li, m, ...)
-       heading | verse content (starting with the first verse class)
-       [ block markup uses 'div', character markup uses 'span' and
-         most blocks are direct children of a 'chapter' ]
-       ...
-     ...
+  chapter
+    label (chapter label)
+    block (e.g. s1, s2, p, li, m, ...)
+      character (e.g. heaing, label, wj, it, note, row)
```
Comment (oparkins) [inline web-admin/yvers-parsing.md:11]: spelling on `heaing`

**Review by oparkins — APPROVED** [review]
I haven't been able to run it yet and my wife wants to not have my attention split at the moment. But I like the new format of the verses -- seems like it will make displaying the verse content less 'magically' and more maintainable. Especially with the style being integrated directly where the text is.

File: web-admin/etc/yvers-parse-version-2 (hunk 1 above)
Comment (depeele) [reply web-admin/etc/yvers-parse-version-2:219]: That's just for debugging and should be commented out if not removed.

(It's the depth to which an object/array will be traversed for display)

File: web-ui/etc/config.yaml (hunk 3 above)
Comment (depeele) [reply web-ui/etc/config.yaml:9]: Yes. You are correct.
I'll make the change.

**Pull request merged**